LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -139,13 +176,7 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-loan-typ-mtg;hasMIGProvider"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-gty;LoanMIGProvider"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-mtg;hasParty"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-pty;LoanServicer"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-typ-mtg;LoanMIGProvider"/>
```

Comment:
No! Not all loans have a MIG e.g. typically if LTV is less than 80%. 
In any case the loan should link to the MIG policy and then the policy to the provider and other info.

Reply:
fixed - made the restriction min 0; also in the case of having a servicer